django/template/loader_tags.py

```diff
@@ -262,6 +258,11 @@ def construct_relative_path(current_template_name, relative_name):
             "same template in which the tag appears."
             % (relative_name, current_template_name)
         )
+    first, last = relative_name[0], relative_name[-1]
+    has_quotes = (
+        (first == '"' and last == '"') or
+        (first == "'" and last == "'")
+    )
```

Comment:
What do you think about using the same conditions as in `ModelAdmin`?
https://github.com/django/django/blob/97e9a84d2746f76a635455c13bd512ea408755ac/django/contrib/admin/options.py#L1035
For example:
```suggestion
    has_quotes = relative_name.startswith(('"', "'")) and relative_name[0] == relative_name[-1]
```

Reply:
I'm fine with that if thats your preference; I was just keeping the intent of the original even as I changed it. Either works, and I'm not super-fussed about relative paths anyway ... I didn't even know they were a thing ;)